internal/publish/database/exposure.go

```diff
@@ -236,6 +237,11 @@ func generateExposureQuery(criteria IterateExposuresCriteria) (string, []interfa
 		q += fmt.Sprintf(" AND traveler = $%d", len(args))
 	}
 
+	if criteria.OnlyNonTravelers {
```

Comment:
we need validation here so as not to have a paradox.

i.e. if you select OnlyTravlers and OnlyNonTravelers

Reply:
Add a check, and added some text to the HTML. PTAL.